script/maven-settings.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <profiles>
+        <profile>
+            <id>apache</id>
+            <repositories>
+	        <repository>
+	            <id>apache-snapshots</id>
+	            <name>Apache Snapshots Repository</name>
+	            <url>https://repository.apache.org/content/repositories/snapshots-group</url>
+	            <releases>
+	                <enabled>false</enabled>
+	                <updatePolicy>never</updatePolicy>
+	            </releases>
+	            <snapshots>
+	                <enabled>true</enabled>
+                    <updatePolicy>always</updatePolicy>
+	            </snapshots>
+	        </repository>
+            <repository>
+	            <id>apache-staging</id>
+	            <name>Apache Staging Repository</name>
+	            <url>https://repository.apache.org/content/repositories/orgapachecamel-1484</url>
```

Comment:
I think the url is wrong.

Reply:
That URL is a placeholder. When a new staging repository is set in the `Makefile` then, it will be updated with the correct value. Likely it's the one related to some test I did to make sure it was changing.